modules/Notes/Note.php

```diff
@@ -205,6 +205,24 @@ public function get_summary_text()
         return "$this->name";
     }
 
+    /**
+     * Returns the content or an empty string if there is no attachment or it
+     * couldn't be located.
+     *
+     * @return string
+     */
+    public function getAttachmentContent()
+    {
+        $path = "upload://{$this->id}";
+        if (!file_exists($path))
+            return '';
+
+        $result = file_get_contents($path);
+        if ($result === FALSE)
```

Comment:
please use PSR standard const names.
see more: https://www.php-fig.org/psr/psr-2/#25-keywords-and-truefalsenull

Reply:
OK, thanks